lib/tasks/learn_to_rank.rake

```diff
@@ -98,7 +99,7 @@ namespace :learn_to_rank do
   end
 
   def assert_ltr!
-    raise "set $ENABLE_LTR to use learn_to_rank" if ENV["ENABLE_LTR"].nil?
+    raise "set $ENABLE_LTR to use learn_to_rank" if !Search::RelevanceHelpers.ltr_enabled?
```

Comment:
This should give a more informative message like `set $ENABLE_LTR to "true"`, as any arbitrary string won't work now.

Reply:
Good point, thanks